app/src/main/java/stage/hoogtebepaling5/MainActivity.java

```diff
@@ -1,91 +1,68 @@
+            handler.postDelayed(new Runnable() {
+                @Override
+                public void run() {
+                    goToLocation(); //after 5 seconds, go to locationclass
+                }
+            }, 5000);
 
-        // checkt of wifi en GPS aanstaan
-        if (wifi.isWifiEnabled() && gps.isProviderEnabled(LocationManager.GPS_PROVIDER)){
-            //laad de plattegrond in scherm
-            ImageView image = (ImageView) findViewById((R.id.plattegrond));
-            image.setImageResource(R.drawable.beganegrond);
-            //image staat standaard uit om bugs te voorkomen, hieronder wordt ie visible gemaakt
-            image.setVisibility(View.VISIBLE);
-            DatabaseAccess databaseAccess = DatabaseAccess.getInstance(this);
-            //Database access openen
-            databaseAccess.open();
-            //query uit getData() wordt in een list geladen
-            List<String> lijst = databaseAccess.getData();
-            //database wordt gesloten
-            databaseAccess.close();
-            //ter controle
-            System.out.println(lijst);
+        }else{
+            goToLocation(); // if wifi is already activated, go to
         }
-        else{
-            //als wifi uitstaat
-            if (!wifi.isWifiEnabled()) {
-
-                //pop up waarin komt te staan dat wifi aanmoet
-                AlertDialog.Builder dialog = new AlertDialog.Builder(MainActivity.this);
-                dialog.setCancelable(true);
-                dialog.setTitle("The app does not work with Wi-Fi turned off");
-                dialog.setMessage("Do you want to turn on Wi-Fi?");
-                dialog.setPositiveButton("Turn me on", (dialog1, id) -> {
-                    wifi.setWifiEnabled(true);
-                })
-                        .setNegativeButton("Cancel ", (dialog12, which) -> {
-                            wifi.setWifiEnabled(false);
-                        });
+    }
 
-                final AlertDialog alert = dialog.create();
-                alert.show();
-            }
-            else{
-                //pop up om gps aan te zetten
-                final Intent action = new Intent(Settings.ACTION_LOCATION_SOURCE_SETTINGS);
-                AlertDialog.Builder dialog = new AlertDialog.Builder(MainActivity.this);
-                dialog.setCancelable(true);
-                dialog.setTitle("The app does not work with Wi-Fi turned off");
-                dialog.setMessage("Do you want to turn on Wi-Fi?");
-                dialog.setPositiveButton("Turn me on", (dialog1, id) -> {
-                    startActivity(action);
-                })
-                        .setNegativeButton("Cancel ", (dialog12, which) -> {
+    public void checkWifi(){
+    final WifiManager wifi = (WifiManager) getApplicationContext().getSystemService(WIFI_SERVICE);
```

Comment:
Je maakt hier een 2e wifiManager aan, moet dat op deze manier of is het handiger om de manager niet in je onCreate te zetten maar gewoon in de class en dan te assignen in je onCreate.

Reply:
You're right. In de class 1x assignen werkt. Heb het net veranderd in de code, kan ik het committen bij de volgende pull of?